### DISCUSSION TITLE
How to set .md template for the location?

### DISCUSSION THREAD

**Discussion by khomutovspace** [datenstrom/community#456] [Ask a question] [answered]
Hello! There is default page-new-default.md and something like this for blog extension. But i can't set it myself for new locations. How to make ir work? For example, i have three locations: magazine, team and blog. And i want to set a new page template for them, so when you are adding page while you are on magazine folder, it has it's own markdown header, and another folder has different. Is there any way to deal with it?

**Comment by markseuffert** [comment, marked as answer]
If I am not mistaken, a new page is based on `Layout` or `LayoutNew`. Here's an example:
```
---
Title: Blog
Layout: blog-start
LayoutNew: blog
---
```

Change `LayoutNew: magazine` and the online editor will use file `page-new-magazine.md`. 

Tested with Datenstrom Yellow 0.8.18. Is this what you're looking for?
